packages/sdks/typescript/src/utils/nodeFetchResponseToReadableStream.ts

```diff
@@ -0,0 +1,19 @@
+import { Readable } from 'stream'
+
+export function nodeFetchResponseToReadableStream(nodeStream: Readable) {
+  return new ReadableStream({
+    start(controller) {
+      nodeStream.on('data', (chunk: Buffer) => {
+        controller.enqueue(chunk)
+      })
+
+      nodeStream.on('end', () => {
+        controller.close()
+      })
+
+      nodeStream.on('error', (err) => {
+        controller.error(err)
+      })
+    },
+  })
+}
```

Comment:
this is very delicate indeed, are there any other events from nodeStream, or only data end and error?

Reply:
This is how body is constructed in `node-fetch` 
https://github.com/node-fetch/node-fetch/blob/main/src/body.js#L37-L89

I don't see any specific. 

This method is typed as `Readable` but is a bit of a lie. If you check in Typescript what node-fetch return is a `NodeJS.ReadableStream`. But that's not a real type a read that is equivalent to Readable
![image](https://github.com/user-attachments/assets/2026f95e-f5a1-4088-bc68-f8d8bd744736)


Here are the events for readable:
https://nodejs.org/api/stream.html#class-streamreadable

Reply:
indeed it was delicate 😂

Reply:
It was very delicate. Still is, not sure I did well. We'll see